data_import/models.py

```diff
@@ -245,6 +252,15 @@ class AWSDataFileAccessLog(models.Model):
     auth_type = models.CharField(max_length=32, null=True)
     host_header = models.CharField(max_length=64, null=True)
 
+    @property
+    def get_data_file(self):
```

Comment:
I'd like this one renamed to `datafile` and it should return a single object or None